src/azul/service/manifest_service.py

```diff
@@ -616,7 +641,16 @@ def entity_type(self) -> str:
         return 'files'
 
     @classmethod
-    def _option(self, s: str):
+    def manifest_properties(cls, url: str) -> JSON:
+        return {
+            'commandLine': {
+                'cmd.exe': None,
+                'bash': f'curl {cls._option(url)} | curl --config -'
```

Comment:
This is missing the point of this entire exercise. Please read the issue description again.

Reply:
The rules for quoting in a curl config file are different to quoting in a shell command line.